packages/client/ui/react-ui/vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+    test: {
+        environment: "jsdom",
```

Comment:
Jest automatically injects a globally `afterEach` into the testing env. The reason we have to add `cleanup()` in each test is because Vitest isn't configured this way by default. After searching online, I found this github issue that explains how we can add the same behaviour by default without needing to call cleanup every test. I'll also link the api docs around this config option:

Fix: 
https://github.com/testing-library/vue-testing-library/issues/296#issue-1577197825

Vitest config option:
https://vitest.dev/config/#globals

Reply:
nice, ty for figuring this out!

Reply:
I added to the config and removed all the calls to cleanup